packages/jest-circus/src/index.js

```diff
@@ -38,8 +38,22 @@ const beforeAll = (fn: HookFn) => _addHook(fn, 'beforeAll');
 const afterEach = (fn: HookFn) => _addHook(fn, 'afterEach');
 const afterAll = (fn: HookFn) => _addHook(fn, 'afterAll');
 
-const test = (testName: TestName, fn?: TestFn) =>
-  dispatch({fn, name: 'add_test', testName});
+const test = (testName: TestName, fn: TestFn) => {
+  if (typeof testName !== 'string') {
+    throw new Error(
+      `Invalid first argument, ${testName}. It must be a string.`,
+    );
+  }
+  if (fn === undefined) {
```

Comment:
It is legal to not pass second argument, I use it as test placeholders.

Might be a a good idea to have an explicit `test.todo` for that case, though